core/internal/cltest/cltest.go

```diff
@@ -533,7 +543,7 @@ func NewEthMocksWithTransactionsOnBlocksAssertions(t testing.TB) *evmclimocks.Cl
 }
 
 // Start starts the chainlink app and registers Stop to clean up at end of test.
-func (ta *TestApplication) Start(ctx context.Context) error {
+func (ta TestApplication) Start(ctx context.Context) error {
```

Comment:
This change seems like a mistake

Reply:
given how the application interface works in the testing this was the simplest change to not require a lot of code changes. Potentially an opportunity for cleanup? 

@jmank88 are there any long term code organization/styling difference son using a pointer receiver vs non-pointer receiver for methods?

Reply:
This is functionally different and likely breaks tests, no? The linter sees the problem on 548.